.github/ISSUE_TEMPLATE/feature-request.yml

```
@@ -0,0 +1,33 @@
+name: Feature Request
+description: Suggest an idea for this project
+title: "[FEATURE]"
+labels: ["enhancement"]
+
+body:
+  - type: textarea
+    id: related-problem
+    attributes:
+      label: "Is your feature request related to a problem? Please describe."
```

Maybe we should indeed rephrase a bit, while at it:

```suggestion
      label: "What kind of feature are you missing? Where do you notice a shortcoming of PTB?"
```

Please make this and the `solution` textbox required